marklogic-data-hub-central/ui-custom/src/components/RecentClear/RecentClear.scss

```diff
@@ -8,4 +8,14 @@
     border: ridge;
     padding: 0px 20px 0px 20px;
   }
+}
+
+.recent-clear-searches {
```

Comment:
I don't know why, but the search Clear button ends up being 22px in height while the records Clear button ends up being 26px in height. Maybe give both buttons an explicit 26px height?